cla_public/static-templates/errors/50x.html

```diff
@@ -6,6 +6,9 @@
     <title>
       {% trans %}Service unavailable - Civil Legal Advice - GOV.UK{% endtrans %}
     </title>
+    <script nonce="{{ csp_nonce() }}" type="text/javascript">
+        (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M9QXCZWK');
+    </script>
```

Comment:
Does this need to be added to a secret; GTM-M9QXCZWK?

Reply:
That's a great question and indeed some people do put the GTM ID in a secret file. But since this ends up visible on the app i.e. in debugging and source code I thought not to add the complexity. So far this has hasn't been an issue.